docs/LocalDevelopment.md

```diff
@@ -24,14 +24,16 @@ Our recommended workflow is to use [`yarn link`][1] to register local `metro` pa
 
     We recommend using `npm exec --workspaces` to register all packages in the `metro` repo — these can be individually linked into the target project later.
 
-        npm exec --workspaces -- yarn link
+    ```bash
```

Comment:
Preferable to use `sh` than `bash` to be consistent with other code blocks in this file